iothub/device/src/ConnectionStatusChangeReason.cs

```diff
@@ -8,13 +8,59 @@ namespace Microsoft.Azure.Devices.Client
     /// </summary>
     public enum ConnectionStatusChangeReason
     {
+        /// <summary>
+        /// This signifies that the client is connected, and ready to be used.
+        /// <para>This connection status change reason is returned with a connection status of <see cref="ConnectionStatus.Connected"/>.</para>
+        /// </summary>
         Connection_Ok,
+
+        /// <summary>
+        /// This signifies that the SAS token associated with the client has expired, and cannot be renewed.
+        /// The supplied credentials need to be fixed before a connection can be established.
+        /// <para>NOTE: This status change reason is currently not used in the client library.</para>
+        /// </summary>
         Expired_SAS_Token,
+
+        /// <summary>
+        /// This signifies that the device/ module has been deleted or marked as disabled (on your hub instance).
```

Comment:
we are trying to use IotHub for consistency everywhere.

Reply:
I can update this to `IoT Hub instance`, sure